pkg/sessiontxn/isolation/repeatable_read.go

```diff
@@ -258,6 +258,11 @@ func (p *PessimisticRRTxnContextProvider) handleAfterPessimisticLockError(ctx co
 			return sessiontxn.ErrorAction(err)
 		}
 	} else if terror.ErrorEqual(kv.ErrWriteConflict, lockErr) {
+		waitTime := time.Since(sessVars.StmtCtx.GetLockWaitStartTime())
+		if waitTime.Milliseconds() >= sessVars.LockWaitTimeout {
```

Comment:
Also check it for [read-committed transactions](https://github.com/pingcap/tidb/blob/c550aae6a125e20c2aaa8d4f11a2f090bc8b2b50/pkg/sessiontxn/isolation/readcommitted.go#L233-L242)?

Reply:
done.